pandera/api/pyspark/model.py

```diff
@@ -300,10 +301,10 @@ def validate(
         random_state: Optional[int] = None,
         lazy: bool = True,
         inplace: bool = False,
-    ) -> Optional[DataFrameBase[TDataFrameModel]]:
+    ) -> Optional[DataFrame[TDataFrameModel]]:
```

Comment:
the pyspark annotation is typed as optional, but the other apis aren't, not sure why, I think there's always a return value?

Reply:
yeah there should always be a return value, the `Optional` should be removed